nvflare/app_opt/xgboost/tree_based/executor.py

```diff
@@ -174,7 +174,9 @@ def _local_boost_bagging(self, fl_ctx: FLContext):
         if self.writer:
             # note: writing auc before current training step, for passed in global model
             self.writer.add_scalar(
-                "AUC", auc, int((self.bst.num_boosted_rounds() - self.num_local_round - 1) / self.num_client_bagging)
+                "train_metrics",
```

Comment:
does this affect existing examples?

Reply:
yes, need to give the same label such that the loaded tensorboard curves will be printed in the same chart